custom-types/react-augment.d.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
instead of disabling the rule, are we able to update to use the React namespace?

```diff
+ ReactElement
- React.ReactElement
```